### PR TITLE
Replace remaining Color(white: opacity:) references with semantic Color assets

### PR DESCRIPTION
**Pull request by lingoslinger**
## Issues covered
https://github.com/planetary-social/nos/issues/1375 (ongoing work)

## Description
Replaced remaining Color(white: opacity:) references with semantically named colors in Colors.xcassets

## How to test
- Verify that the white: opacity: colors in each asset match the hard coded white: opacity: color in the changed file; smoke test the app to verify

## Notes
- changes to  hard coded RGB colors will be addressed in a separate PR.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -11,7 +11,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Removed the like and repost counts from the Main and Profile feeds.
 - Fixed an issue where the sheet asking users to set up a NIP-05 username would appear after reinstalling Nos, even if the profile already had a NIP-05 username.
 - Fixed a bug where urls with periods after them would include the period.
-- Replaced hard-coded color values.
+- Replaced hard-coded hex and white: opacity: color values.
```
Comment (joshuatbrown) [inline CHANGELOG.md:14]: I think the original is better -- we don't need so much detail in the CHANGELOG.
```suggestion
- Replaced hard-coded color values.
```

File: Nos/Views/Form/NosFormSection.swift [hunk 2]
```diff
@@ -32,7 +32,7 @@ struct NosFormSection<Content: View>: View {
                 .cornerRadius(21)
                 .offset(y: 4.5)
                 .shadow(
-                    color: Color(white: 0, opacity: 0.2), 
+                    color: Color.nosFormSectionShadow,
```
Comment (joshuatbrown) [inline Nos/Views/Form/NosFormSection.swift:35]: This is the same color as `bigActionButtonShadow`, so I'd remove one and name the other more generically so we don't have to duplicate the color. Something like `light-shadow` in the Colors.xcassets.

**Review by joshuatbrown — CHANGES_REQUESTED** [review]
This is a partial review; I'll do a full one when #1403 is merged so the diff here is smaller. A couple things to change for now.

File: CHANGELOG.md (hunk 1 above)
Comment (lingoslinger) [reply CHANGELOG.md:14]: Understood, I will change this in the final PR for RGB values.

File: Nos/Views/Form/NosFormSection.swift (hunk 2 above)
Comment (lingoslinger) [reply Nos/Views/Form/NosFormSection.swift:35]: Was not sure how you wanted duplicates handled; now I know. 🙂

File: CHANGELOG.md (hunk 1 above)
Comment (joshuatbrown) [reply CHANGELOG.md:14]: Let's change it now and skip the changelog update in the future PRs.

**Review by joshuatbrown — CHANGES_REQUESTED** [review]
One more request then we're all good -- I tested and everything else looks great! Thanks again!

**Review by joshuatbrown — APPROVED** [review]
Thanks again for your work on this! The code is much nicer this way!

**Pull request merged**